app_dart/lib/src/request_handlers/push_engine_status_to_github.dart

```diff
@@ -84,8 +84,7 @@ class PushEngineStatusToGithub extends ApiRequestHandler<Body> {
         request.targetUrl = 'https://ci.chromium.org/p/flutter/g/engine/console';
         request.context = 'luci-engine';
         if (status != GithubBuildStatusUpdate.statusSuccess) {
-          request.description = 'Flutter build is currently broken. Please do not merge this '
-              'PR unless it contains a fix to the broken build.';
+          request.description = config.flutterBuildDescription;
```

Comment:
flutterBuildDescription always going to be populated, should we have a default value?

Reply:
By default, it will be empty not showing a message. I believe GitHub defaults to "Success" in that case.